css/style.css

```diff
@@ -0,0 +1,1002 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato&display=swap");
+@import url("https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.3/font/bootstrap-icons.css");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  scroll-behavior: smooth;
+}
+
+body {
+  overflow-x: hidden;
+  margin: auto;
+  height: auto;
+  width: auto;
+}
+
+header {
+  display: flex;
```

Comment:
- [ ] Kindly consider creating a class name flex like :
```
.flex {
 display: flex;
} 
```
and then just give `flex` class to all the elements that needs flex to make your code DRY.
You can use this technique to for other properties to make your CSS code dry and clean.